e2e_tests/tests/cluster/test_users.py

```diff
@@ -351,60 +396,61 @@ def test_experiment_creation_and_listing(clean_auth: None) -> None:
 
 
 @pytest.mark.e2e_cpu
-def test_login_wrong_password(clean_auth: None) -> None:
-    creds = create_test_user(ADMIN_CREDENTIALS, True)
+def test_login_wrong_password(clean_auth: None, login_admin: None) -> None:
+    creds = create_test_user(True)
 
     passwd_prompt = f"Password for user '{creds.username}':"
-    unauth_error = r".*Forbidden\(invalid credentials\).*"
-
     child = det_spawn(["user", "login", creds.username])
     child.setecho(True)
     child.expect(passwd_prompt, timeout=EXPECT_TIMEOUT)
     child.sendline("this is the wrong password")
-
-    child.expect(unauth_error, timeout=EXPECT_TIMEOUT)
-    child.read()
+    unauth_error = "invalid credentials"
+    assert unauth_error in str(child.read())
     child.wait()
     child.close()
 
     assert child.exitstatus != 0
 
 
 @pytest.mark.e2e_cpu
-def test_login_as_non_existent_user(clean_auth: None) -> None:
+def test_login_as_non_existent_user(clean_auth: None, login_admin: None) -> None:
     username = "doesNotExist"
 
     passwd_prompt = f"Password for user '{username}':"
-    unauth_error = r".*Forbidden\(user not found\).*"
+    unauth_error = "invalid credentials"
 
     child = det_spawn(["user", "login", username])
     child.setecho(True)
     child.expect(passwd_prompt, timeout=EXPECT_TIMEOUT)
     child.sendline("secret")
 
-    child.expect(unauth_error, timeout=EXPECT_TIMEOUT)
-    child.read()
+    assert unauth_error in str(child.read())
     child.wait()
     child.close()
 
     assert child.exitstatus != 0
 
 
 @pytest.mark.e2e_cpu
-def test_login_with_environment_variables(clean_auth: None) -> None:
-    creds = create_test_user(ADMIN_CREDENTIALS, True)
+def test_login_with_environment_variables(clean_auth: None, login_admin: None) -> None:
+    creds = create_test_user(True)
+    # logout admin
+    assert log_out_user() == 0
```

Comment:
it seems a bit annoying to do `assert log_in_user() == 0` and `assert log_out_user() == 0` every time. we pretty much never expect it to fail.

maybe instead of having these two helpers return exit code which needs to be checked every time, have them throw an exception on errors instead? then this would just become simple `log_out_user()` line.